compass_sdk/__init__.py

```diff
@@ -285,14 +300,14 @@ class ParserConfig(ValidatedModel):
     min_chars_per_element: int = DEFAULT_MIN_CHARS_PER_ELEMENT
     skip_infer_table_types: List[str] = SKIP_INFER_TABLE_TYPES
     parsing_strategy: ParsingStrategy = ParsingStrategy.Fast
-    parsing_model: ParsingModel = ParsingModel.Marker
+    parsing_model: ParsingModel = ParsingModel.YoloX_Quantized
```

Comment:
Any specific reason for changing the strategy from unstructured?

Reply:
Hmm that's weird... the default strategy should have never been `Marker`. We don't even have `Marker` as an option in the `compass-parser`. I think it still worked because the `parsing_model` doesn't have the value in the parser side and missing values default to unstructured. Anyway, we don't use unstructured anymore for PDF.